app/Http/Livewire/Sport/SportStandingForm.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Livewire\Sport;
+
+use App\Actions\Sport\SubmitSportStandingAction;
+use App\Models\Country;
+use App\Models\CountrySport;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Support\Collection as SupportCollection;
+use Illuminate\Support\Facades\Cache;
+use Livewire\Component;
+
+class SportStandingForm extends Component
+{
+    public $sport;
+
+    /**
+     * State of this component
+     *
+     * @var array
+     */
+    public $state;
+
+    public function mount($sport, $state): void
+    {
+        $this->sport = $sport->withoutRelations();
+        $this->defaultState($state);
+    }
+
+    public function getCountriesProperty()
+    {
+        /**
+         * Using cache for optimizing get countries data each component is rendered
```

Comment:
👍

Reply:
I think that is the best way can i think because if i try to pass the countries data from controller and than make any changes on component, Livewire will execute new query using `where id in` statement. That make me feel weird because actually that is just getting all data.